src/Raven.Server/NotificationCenter/NotificationsStorage.cs

```diff
@@ -344,9 +332,11 @@ public string GetDatabaseFor(string id)
         public void ChangePostponeDate(string id, DateTime? postponeUntil)
         {
             using (_contextPool.AllocateOperationContext(out TransactionOperationContext context))
-            using (var tx = context.OpenWriteTransaction())
             {
-                using (var item = Get(id, context, tx))
+                var getCommand = new GetNotificationCommand(id, this);
```

Comment:
There is no need for a command for a get.
If you want to use everything in the same transaction than you can put everything in the `StoreNotificationCommand`.